core-domain/src/test/java/org/uniprot/core/uniprotkb/comment/impl/DiseaseCommentBuilderTest.java

```diff
@@ -96,7 +95,8 @@ public static Comment createObject() {
         List<EvidencedValue> evidencedValues = new ArrayList<>();
         evidencedValues.add(
                 new EvidencedValueBuilder("value 1", EvidenceBuilderTest.createObjects(3)).build());
-        evidencedValues.add(new EvidencedValueBuilder("value2", emptyList()).build());
+        evidencedValues.add(
```

Comment:
There is no specific reason. Since it(EvidencedValue and Evidence) is a type of List so i have added more than one item. I changed the `emptyList` to `EvidenceBuilderTest.createObjects(2)` because call `ValidateJson.verifyEmptyFields` was failing since it was empty.